Example/AppDelegate+CarPlay.swift

```diff
@@ -99,6 +108,46 @@ extension AppDelegate: CarPlayManagerDelegate {
             favoriteTemplateButton.image = UIImage(named: "carplay_star", in: nil, compatibleWith: traitCollection)
             return [favoriteTemplateButton]
         case .navigating:
+            let exitButton = CPBarButton(type: .text) { _ in
+                carPlayManager.currentNavigator?.exitNavigation(byCanceling: true)
+            }
+            exitButton.title = "End"
+            return [exitButton]
+        }
+    }
+    
+    func carPlayManager(_ carPlayManager: CarPlayManager, mapButtonsCompatibleWith traitCollection: UITraitCollection, in template: CPTemplate, for activity: CarPlayActivity, carPlayMapViewController: CarPlayMapViewController) -> [CPMapButton]? {
+        
+        switch activity {
+        case .navigating:
+            let showFeedbackButton = CPMapButton { button in
+                carPlayManager.currentNavigator?.showFeedback()
+            }
+            showFeedbackButton.image = UIImage(named: "carplay_feedback", in: nil, compatibleWith: nil)
+            
+            let overviewButton = CPMapButton { button in
```

Comment:
Could we extract all of this other button creation into `createSuchAndSuchButton(for: dependency)` functions?

Reply:
Posterity: after talking with Kobe about this, it seems like punting is the right choice.